backend-project/small_eod/letters/migrations/0012_auto_20200626_0848.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.0.6 on 2020-06-26 08:48
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('letters', '0011_auto_20200618_1921'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='letter',
+            name='identifier',
+        ),
+        migrations.AddField(
```

Comment:
na chwilę obecną nas to nie boli (bo nie mamy  żadnego środowiska, gdzie utrzymanie danych jest znaczące, a jak będziemy mieli to migracje zostaną zeskłoszowane), ale docelowo warto zwracać uwagę co się dzieje w migracjach - tutaj, zamiast sekwencji `RemoveField` i `AddField` powinniśmy użyć po prostu `RenameField` - inaczej byśmy stracili zawartość `identifier` i `reference_number` posiadałby tylko`NULL`

Reply:
ok, będę pamiętał.